models/tokens/tokens_erc20_legacy.sql

```diff
@@ -3,7 +3,7 @@
         post_hook='{{ expose_spells(\'["arbitrum","avalanche_c","bnb","ethereum","optimism", "gnosis", "fantom", "polygon"]\',
                                     "sector",
                                     "tokens",
-                                    \'["0xManny","hildobby","soispoke","dot2dotseurat","mtitus6"]\') }}')}}
+                                    \'["0xManny","hildobby","soispoke","dot2dotseurat","mtitus6", "denver"]\') }}')}}
```

Comment:
can you also remove your name here? then add it to the non-legacy file instead? it'd be good to see this file out of the PR altogether

Reply:
absolutely, missed this on the first pass!